src/pudl/analysis/service_territory.py

```diff
@@ -384,14 +383,14 @@ def _limit_years(df: pd.DataFrame) -> pd.DataFrame:
     return geom
 
 
-def compiled_geoms_asset_factory(
-    entity_type: Literal["ba", "util"],
+def service_territory_asset_factory(
+    entity_type: Literal["balancing_authority", "utility"],
```

Comment:
We aren't actually storing any geometry information in the table (though we could, if we wanted to use GeoParquet and/or SpatiaLite) so naming these things "geometry" seemed misleading.